Makefile

```diff
@@ -40,23 +40,24 @@ deps:
 	go generate -tags tools tools/tools.go
 
 .PHONY: format
-format: fmt vet errcheck imports
+format:
```

Comment:
The idea behind format was to handle all formatting with `make format` if you want to reduce the commands run with int-test down to just `gofumpt` then what if we just changed the `int-test` and `unit-test` target to point at `fmt-check` instead of `format`?